src/ifk_scb_compilations/objects/emissions_kommun.py

```diff
@@ -0,0 +1,98 @@
+"""Fetch emission data by kommun.
+
+Observera att i tabellen över utsläpp från transporter ingår för närvarande endast vägtransporter.
+Miljöräkenskapernas statistik om utsläpp till luft utgår från ett produktionsperspektiv och redovisar därför
+direkta utsläpp från svenska ekonomiska aktörer. Produktionsperspektivet är annorlunda från det territoriella
+perspektivet (som används för att rapportera Sveriges utsläpp till FN), vilket avgränsas till utsläpp som
+sker inom Sveriges gränser. Samtidigt används ett territoriellt perspektiv i mycket av den underlagsdata
+som används för att producera miljöräkenskapernas statistik om utsläpp till luft. För att justera för
+skillnaden mellan miljöräkenskapernas produktionsperspektiv och det territoriella perspektivet används
+en så kallad residensjustering. Residensjusteringen appliceras framförallt på transportrelaterade utsläpp.
+Mer information om beräkningsmetodiken och underlagsdata som används för residensjusteringen redovisas i
+statistikens Kvalitetsdeklarationen.
+"""
+
+
+import pandas as pd
+from pyscbwrapper import SCB
+
+
+class FetchData:
+    """Dataclass for emissions by kommun and year."""
+
+    def __init__(self, emission_type="växthusgaser, kiloton koldioxidekvivalenter"):
+        """Initialization."""
+        self.emission_type = emission_type
+        self.query = ["MI", "MI1301", "MI1301B", "UtslappKommun"]
+        self.scb = SCB("sv")
+        self.scb.go_down(*self.query)
+        self.region_id = self.scb.info()["variables"][0]["values"]
+        self.regioner = self.scb.info()["variables"][0]["valueTexts"]
+        self.years = self.scb.info()["variables"][3]["values"]
+
+    def fetchData(self) -> dict:
```

Comment:
```suggestion
    def fetch_data(self) -> dict:
```

A thought. I guess one could think that the user will instantiate and call this object as 

```python
fdata = FetchData()
fdata.get_data()
```

but it might be enough with

```python
fdata.get()
```

Reply:
updated.